src/FurCoNZ.Web/Auth/NzFursOpenIdConnectEvents.cs

```diff
@@ -17,37 +20,40 @@ public static class NzFursOpenIdConnectEvents
         public static async Task OnUserInformationReceived(UserInformationReceivedContext context)
         {
             var userService = context.HttpContext.RequestServices.GetService<IUserService>();
+            var linkGenerator = context.HttpContext.RequestServices.GetService<LinkGenerator>();
             var identity = (ClaimsIdentity)context.Principal.Identity;
             var user = await userService.GetUserFromIssuerAsync(context.Principal.FindFirst("iss").Value, context.Principal.FindFirst("sub").Value, context.HttpContext.RequestAborted);
 
             // User already exists,m nothing to do?
```

Comment:
Really minor typo in existing code:
```suggestion
            // User already exists, nothing to do?
```